admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,193 @@
+==========================
+Configuring Memory Caching
+==========================
+
+You can significantly improve your ownCloud server performance with memory 
+caching, where frequently-requested objects are stored in memory for faster 
+retrieval. There are two types of caches to use: PHP opcache, and data caching 
```

Comment:
> There's a module called Memcache

True, but opcache is a combined word for "opcode cache". And memcache is the correct term. And we should clearly say here that it is a cache for opcode (general term) and not opcache (specific implementation).